.github/workflows/CI.yaml

```diff
@@ -0,0 +1,26 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - trunk
+  pull_request:
+    branches:
+      - trunk
+
+jobs:
+  ci:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: clean install
+        run: npm ci
+      - name: build
```

Comment:
We don't need to build before calling build:staging, maybe instead use a matrix to test a build:staging and a build:production in parallel?